.github/workflows/main.yml

```diff
@@ -5,8 +5,14 @@ on: [push, pull_request]
 env:
   DEVELOPER: 1
 
+permissions:
+  contents: read
+
 jobs:
   ci-config:
+    permissions:
+      contents: read
+      actions: read # for github.actions.getWorkflowRun
```

Comment:
Did you mean to add a comment after the `#`?

Reply:
Yes